index.html

```diff
@@ -6,11 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="/style.css">
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    
+    <main>
+        <nav>
```

Comment:
Good job on remembering to use semantic element `main` for your content however your `nav` should live in the `header` as it is always part of the top of your page.